Gemfile

```diff
@@ -5,7 +5,11 @@ ruby '2.6.1'
 gem 'jekyll'
 gem 'json'
 gem 'hash-joiner'
-gem 'uswds-jekyll'
+gem 'uswds-jekyll', git: 'https://github.com/18F/uswds-jekyll', ref: 'refs/pull/182/head'
+
+# workaround to support _pages collection
+# https://github.com/jekyll/jekyll-sass-converter/issues/93#issuecomment-524270623
+gem 'jekyll-sass-converter', git: 'https://github.com/jekyll/jekyll-sass-converter.git', ref: 'refs/pull/94/head'
```

Comment:
We can switch to released versions of those gems once https://github.com/18F/uswds-jekyll/pull/182 and https://github.com/jekyll/jekyll-sass-converter/pull/94 are merged, respectively.

Reply:
Aidan, are you wanting to hold this until then?